applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -31,7 +31,7 @@ class PotentialFlowTests(UnitTest.TestCase):
 
     def setUp(self):
         # Set to true to get post-process files for the test
-        self.print_output = False
+        self.print_output = True
```

Comment:
```suggestion
        self.print_output = False
```
this should only be used for debgging, not for regular test executions

Reply:
Thank you for the suggestions! I will resolve these issues